lib/jekyll-relative-links/generator.rb

```diff
@@ -21,16 +21,52 @@ def initialize(site)
     end
 
     def generate(site)
+      # Get any configuration specified
+      # Defaults mean no change in behaviour
+      jrl_config = site.config["jekyll_relative_links"] || {}
+      @config = {}
+      @config["collections"] = jrl_config["collections"] || []
+      @config["process_all_collections"] = jrl_config["process_all_collections"] || false
+      @config["verbose"] = jrl_config["verbose"] || 0
```

Comment:
Fair enough. I had to look up what "memoized" meant but, since you've used that approach already, that would make sense. I think you have a clearer idea of how you would like to see this implemented than I do, though. I can understand the "memoized" bit now, but "stored in a class-level constant hash" is leaving me a bit blank ...